src/utils/temp_folder.test.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai';
+import { cleanUpTempFolder, getTempFolder } from './temp_folder';
+import * as fs from 'fs';
+
+describe('temp folder test', () => {
+	it('temp returns a folder that exists', () => {
+		const tempFolderPath = getTempFolder();
+		expect(fs.existsSync(tempFolderPath)).to.equal(true);
+	});
+
+	it('temp path returned contains the correct subfolders', () => {
+		const tempFolderPath = getTempFolder();
+		expect(tempFolderPath).to.contains('ardrive');
+		expect(tempFolderPath).to.contains('temp');
+	});
+
```

Comment:
Please add a test verifying that getTempFolder() can be called twice in a row without error.